### PR TITLE
Add July team meeting minutes

### LINKED ISSUE #308: Hubs Team Meeting - July 2020
Hello everyone!

Reminder that our next meeting is **Thursday, July 16th, 2020 at 5PM UTC** (or your [timezone](https://arewemeetingyet.com/UTC/2020-07-16/17:00/Hubs%20Team%20Meeting)). 

* Agenda - HackMD: https://hackmd.io/@sgibson91/hubs-team-meeting
* Video - Google Meet: https://meet.google.com/dwh-uujk-ofi

These calls are open to **anyone**, so please join us! We love seeing new members on the call 😄

### PR DESCRIPTION
**Pull request by sgibson91**
This PR adds the minutes from the July team meeting and also addresses some points raised by @choldgraf in https://github.com/jupyterhub/team-compass/issues/308#issuecomment-659074261

fixes #308

### CODE REVIEW

**Comment by rkdarst** [comment]
tldr `<https://example.com>` is commonmark syntax for automatic links.

I just checked one of my sites, and it seems that rst links are automatically detected (so it's not sphinx but the parser).  In markdown, I recently learned that `<https://example.com>` is needed.  But since markdown mutually compatible, I guess github markdown automatically does it, but not commonmark / the recommonmark parser uses.  It seems hackmd doesn't require <>, but I think jekyll does...

I don't promise that something above is 100% correct, though...

**Comment by choldgraf** [comment]
yep - that's correct. CommonMark doesn't have any special syntax for URLs, while GFM has regexes for "things that start with `http` to auto-link them.

**Comment by sgibson91** [comment]
I added `<>` to the bare URLs, hopefully they'll become clickable!

**Comment by manics** [comment]
💯  https://jupyterhub-team-compass--318.org.readthedocs.build/en/318/monthly-meeting/2020-07-16.html

**Comment by choldgraf** [comment]
wohoo, thanks @sgibson91 !
